pygments/lexers/crystal.py

```diff
@@ -171,6 +171,7 @@ def gen_crystalstrings_rules():
             '''.split(), suffix=r'\b'), Keyword),
             (words('''
                 previous_def forall out uninitialized __DIR__ __FILE__ __LINE__
+                __END_LINE__
```

Comment:
The way that the lines are split looks weird, but this is technically the most correct way, so nothing to do here.

Reply:
I'm not familiar with code style of this project, but all lines seem to be limited within 90 characters. If this is put at L173, L173 will exceed 90 chars. Please let me know if my understanding is not correct.

Reply:
Our "desired" target line length is 90 characters, so this looks fine. The 90 character rule hasn't been enforced but it's a general guideline.